README.md

```diff
@@ -23,3 +23,60 @@ Then,
 
 `ansible-playbook -i inventory/hosts webscale.yml`
 
+## BigIP Pre-requisites
+
+Tested for - OCP 4.5.11
+
+1.  Need at least 1 worker node to host bigip Virtual machines
+2.  Install SRIOV and OSV operators in the cluster, so obviously Hardware must support SRIOV.
```

Comment:
Should we integrate your ansible playbook to enable SR-IOV at some point into this repo?

Reply:
yes, we can import `bigip-setup.yml` playbook in to `webscale.yml` playbook file later on.